plugins/lando-services/services/mariadb/builder.js

```diff
@@ -8,13 +8,13 @@ module.exports = {
   name: 'mariadb',
   config: {
     version: '10.3',
-    supported: ['10.5', '10.4', '10.3', '10.2', '10.1'],
+    supported: ['10.6', '10.5', '10.4', '10.3', '10.2'],
```

Comment:
Currently we don't remove support completely for deprecated versions of services, we simply move them to a "legacy" array (see https://github.com/lando/cli/blob/main/plugins/lando-services/services/php/builder.js#L96 as an example).

Reply:
...good confirmation of this is that 10.1 was the most-used mariadb service last month.